clouddriver-api/clouddriver-api.gradle

```diff
@@ -21,6 +21,7 @@ dependencies {
   annotationProcessor platform("com.netflix.spinnaker.kork:kork-bom:$korkVersion")
 
   api("com.netflix.spinnaker.kork:kork-plugins-api")
+  api("com.netflix.spinnaker.kork:kork-credentials")
```

Comment:
No can do. `api` modules must be _very_ limited in the dependency graph that it exposes: `kork-credentials` has a dependency on Spring. If you want to expose `kork-credentials`, you'll need to remove the Spring dependency, e.g. by creating a `kork-credentials-spring` or similar that is included in services, but not included within `api` modules.

Reply:
I split `kork-credentials-api` here: https://github.com/spinnaker/kork/pull/787. Will update that PR afterwards.

Reply:
Updated with `clouddriver-api` -> `kork-credentials-api`, `clouddriver-core` -> `kork-credentials`. Marking as resolved.